src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,63 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Page Block Magnification
+
+Page blocks have a magnification option that allows the user to zoom in order to inspect the details of the block in distraction-free mode.
```

Comment:
`the user to zoom in order to inspect`
I find this combination of words a bit tricky to read; try to reword it